public/video-ui/src/components/Videos/ComposerPageCreate.js

```diff
@@ -12,10 +12,22 @@ export default class ComposerPageCreate extends React.Component {
     return getStore().getState().config.composerUrl;
   };
 
+  getComposerId = () => {
+    const usages = getStore().getState().usage.data;
+    if (usages.preview.video.length && !usages.published.video.length) {
```

Comment:
IIUC if the Composer page has been published, we will fail?

Maybe we update the [usage response](https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/services/VideosApi.js#L134-L145) or check `if (totalVideoPages > 0)`?

Reply:
even better, we could do something like:

```js
const videoPages = [...usages.preview.video, ...usages.published.video]
if (videoPages.length !== 0) {
  return videoPages.[0].fields.internalComposerCode
}
```